pkg/stanza/fileconsumer/file_test.go

```diff
@@ -498,6 +498,8 @@ func TestReadNewLogs(t *testing.T) {
 
 	// Expect the message to come through
 	waitForToken(t, emitCalls, []byte("testlog"))
+
+	operator.closeFiles()
```

Comment:
is it not worth using t.Cleanup for this?

Reply:
Great idea. This simplified it quite a lot because it can be called from the `buildTestManager` function.

Reply:
This is updated and passed CI